### PR TITLE
span:indent supports percent for fix and break mode

### PR DESCRIPTION
**Pull request by guoweilkd**
### Description of the feature or fix

indent supports percent for fix and break mode.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

**Comment by kisvegabor** [comment]
Thank you!

**Comment by kisvegabor** [comment]
Doesn't it require to update the docs?

**Comment by guoweilkd** [comment]
> Doesn't it require to update the docs?

done

File: docs/widgets/extra/span.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The spangroup can be set to one the following modes:
 Use `lv_spangroup_set_overflow(spangroup, LV_SPAN_OVERFLOW_CLIP)` to set object overflow mode.
 
 ### first line indent
-Use `lv_spangroup_set_indent(spangroup, 20)` to set the indent of the first line, in pixels.
+Use `lv_spangroup_set_indent(spangroup, 20)` to set the indent of the first line.all modes support pixel units, in addition to LV_SPAN_MODE_FIXED and LV_SPAN_MODE_BREAK mode supports percentage units.
```
Comment (kisvegabor) [inline docs/widgets/extra/span.md]:
```suggestion
Use `lv_spangroup_set_indent(spangroup, 20)` to set the indent of the first line. all modes support pixel units, in addition to LV_SPAN_MODE_FIXED and LV_SPAN_MODE_BREAK mode supports percentage units too.
```

**Comment by kisvegabor** [comment]
Thank you. I've pushed a minor fix to the docs.

**Pull request merged**